.github/mergify.yml

```
@@ -23,6 +23,24 @@ pull_request_rules:
     actions:
       queue:
 
+  - name: Conventional Commit PR title
+    conditions:
+      - base=master
+    actions:
+      post_check:
+        success_conditions:
+          - "title~=^(fix|feat|docs|refactor|chore|deps)(?:\\(.+\\))?!?:"
```

```suggestion
          - "title~=^(fix|feat|docs|refactor|chore|deps)(\\(.+\\))?!?:"
```

The `:` at the start of the scope is wrong IMO.

---

I tested the new one here: https://www.regexpal.com/

In case you want to reproduce it, make sure to remove the escaping of `\` which is only necessary in this config file because the expression is within a string. (I think?)